### ISSUE TITLE
work on figure 2 -- levels of diversity

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/asevdiversitydup#2]
[Figure 1](https://github.com/lizzieinvancouver/asevdiversitydup/blob/main/figures/scan7May2023vitis.png)

I am not 100% sure we will keep this figure so work on it quickly! Minimal time ....

It is supposed to be a map of the northern hemisphere (find a bland creative commons one or make one in R) that fades out at the top and bottom and lists the number of species of Vitis on each continent. Then has something vaguely looking like a coalescent plot ([see first figure here](http://winegourd.blogspot.com/2018/11/the-genealogy-of-grapes.html)) of varieties and clones. There really should be a ... as we cannot show all relationships. 

Again, work on this quickly without too much effort.

**Comment by christophe-rd** [comment]
Perfect, that sounds good! I haven't spent that much time on the figure (gave me the occasion to learn Illustrator). I think I pushed it properly in the repo. Could you please let me know if that is correct? 
*Also is it ok if I answer here?

**Comment by lizzieinvancouver** [comment]
@christophe-rd Yes! It's great if you answer as managing git issues is much easier for me than trying to keep track of my inbox. 

The figure pushes look good to me. You can check them from clicking on each [here](https://github.com/lizzieinvancouver/asevdiversitydup/commits/main); then you can confirm what changed etc.

For figure 1, it looks good! I suggest making the periods smaller or maybe putting them within boxes as is often done (see Fig 1 [here](https://www.pnas.org/doi/10.1073/pnas.1809600115) for an example) and use a sans serif font for figures; I like Helvetica). 

If you can push png drafts (from Illustrator, file menu then 'export' does this fairly quickly) it would be easier for me. 

Thanks!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Sorry, get lost in figures ... I updated above. For this figure, it looks good! As the focus is supposed to be on the diversity levels (all the lines within Europe), can you crop down the map to northern hemisphere (and fade the edges if easy; google 'fade to transparent illustrator') and make the levels of diversity coming from Europe much bigger then I think this will be close to a good first draft.

Thanks for your help on this!

**Comment by christophe-rd** [comment]
Perfect, I found a way to fade the map to transparent. I pushed it into the repo!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Looks great! I have a few requests ... 

- [ ] Make the background map fainter 
- [ ] Add some names to the varieties. For simplicity I would pick the first A from [here](https://plantgrape.plantnet-project.org/en/cepages) and the last Z variety and just label three dots: first, last and the one that matches with a rootstock.  I suggest we label that one Chardonnay or move it later and label Pinot noir. 
- [ ] Ditto for the clones, but then just label first and last (they are often numbers, if you click on any variety in last link you should see a list under 'Clonal selection in France')
- [ ] And pick an example rootstock for the label! [SO4 is common](https://plantgrape.plantnet-project.org/en/porte-greffe/S%C3%A9lection%20Oppenheim%204) but you can pick whatever you like. You likely need to edit the figure to show the other North American species that is crossed (e.g., Vitis berlandieri and Vitis riparia) to produce the rootstock, then link it to the variety (scion)

Eventually we should find a less pixelated map (I usually build one in R) but let's wait until we see what co-authors think.

**Comment by christophe-rd** [comment]
Sorry again for not answering you this week!  I need to dedicate additional time to address the faded edges properly. I will keep you updated on my progress.

**Comment by christophe-rd** [comment]
Hi Lizzie!
I worked on the things you requested:
- I made the background map look fainter. Is that enough?
- I did add some variety. I skipped the first variety (Abondant B) since it didn't have any clones. I chose instead Abouriou N and 603 is a clone of that variety. Idem for Xinomavro/1255. They are not really related in the graph, but I figured it would be more simple for now to stick with the clones of the mentioned varieties.
- Is Chardonnay at the right place?
- I added arrows for clarity, I can change them if you prefer a different style
- For the rootstock, I kept the older version with RX5 as I didn't fully understand your suggestion about producing the rootstock and linking it to the variety
- The map should be a less pixelated map now. It's an SVG.

**Comment by lizzieinvancouver** [comment]
@christophe-rd Thanks!

- Fading and background look great
- I really like how did the _riparia_ x _berlandia_ to SO4 -- nice!
- For the RX5 ... What I wanted to show is that the roostock ends up on a scion which is one clone of a variety. Maybe for now we can show that by deleting 'RX5 plants' and extend the arrow from SO4 to a clone (and write next to the arrow 'rootstock for _vinifera_ scion' 

The way the pedigree works, we should shown the clones coming from a specific variety. So, can you relabel the variety dot that produces the clones as [Chardonnay](https://plantgrape.plantnet-project.org/en/cepage/Chardonnay) (you could put the name underneatand relabel the clones as 75 at the start and 1147 at the end? ... Wait, this is more confusing to type than to do, so I did it in AI and pushed an update. I hope I did not ruin all your groupings.

**Comment by christophe-rd** [comment]
Perfect! Sorry again for my late response and thank you for your comments. 
>For the RX5 ... What I wanted to show is that the roostock ends up on a scion which is one clone of a variety. Maybe for now we can show that by deleting 'RX5 plants' and extend the arrow from SO4 to a clone (and write next to the arrow 'rootstock for vinifera scion'

I pushed back the new version with RX5 deleted. Could you please confirm that this is what you imagined?

If you need me to modify anything else, please let me know!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Yes, this looks great! I am waiting to hear back from a colleague (will probably be a couple weeks) then should have a better sense of how we want to finalize figures.

**Comment by christophe-rd** [comment]
> @christophe-rd Yes, this looks great! I am waiting to hear back from a colleague (will probably be a couple weeks) then should have a better sense of how we want to finalize figures.

Perfect, thanks for the feedback!

**Comment by lizzieinvancouver** [comment]
@christophe-rd I haven't been asking for tweaks on this as Andy Walker suggested cutting it, but Iñaki wants to keep it. Unless Thierry feels strongly I think we will keep it, but we can wait to hear from him. However if you're girlfriend has any suggestions on it, that would be most welcome.

**Comment by christophe-rd** [comment]
@lizzieinvancouver perfect! I will ask her for suggestions and will let you know! Thanks!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Iñaki was confused on Fig 2 by what Rx5 meant. I confess I cannot recall either, but I am sure we can fix it! Remind me and we can hopefully come up with a way to clarify it.

**Comment by christophe-rd** [comment]
Hi! I believe it's an older version of the figure that is currently in the ms. [Here's the link of the last version ](https://github.com/lizzieinvancouver/asevdiversitydup/blob/284875d0834328202dd277b90c33ba2215577ab6/figures/figure_1.png)

**Comment by christophe-rd** [comment]
@lizzieinvancouver [here's](https://github.com/lizzieinvancouver/asevdiversitydup/blob/f4f150a75138d3884ae431ebf05dfc899933c236/figures/figure_1.png) an updated version of the figure with some minor adjustments on the titles. I made all the formats uniform across the 3 different figures.

**Comment by lizzieinvancouver** [comment]
@christophe-rd Re: 

> I don't know the convention for refs in a figure. Is it appropriate to add these references here? If so, should we also include them in the references at the end of the paper

Good point! Yes, I can add them to the ref list. If you can find them in Google Scholar then you can save them and export to bibtex and I can import them. Or just send me the full citation info and I can easily do it.

**Comment by lizzieinvancouver** [comment]
@christophe-rd Can you make the fonts overall larger? I think they are a little hard to read now.

**Comment by christophe-rd** [comment]
> Good point! Yes, I can add them to the ref list. If you can find them in Google Scholar then you can save them and export to bibtex and I can import them. Or just send me the full citation info and I can easily do it.

Perfect, you I created a folder in the figure folder. You can find them both [here](https://github.com/lizzieinvancouver/asevdiversitydup/tree/60fd55dd35be8ca9e0a1d1202385122f28c9b42c/figures/references_for_fig_1). I couldn't find the references I mentioned in the figure, but I think the information was still relevant. I changed the articles where the info came from. 

>@christophe-rd Can you make the fonts overall larger? I think they are a little hard to read now.

Yes, I made the fonts larger! [Could you please tell me if it's better now?](https://github.com/lizzieinvancouver/asevdiversitydup/blob/60fd55dd35be8ca9e0a1d1202385122f28c9b42c/figures/figure_1.png)

**Comment by lizzieinvancouver** [comment]
@christophe-rd Looks great! Thank you!
